thefuck/rules/cd_mkdir.py

```diff
@@ -11,11 +10,13 @@ def match(command):
         command.script.startswith('cd ') and any((
             'no such file or directory' in command.output.lower(),
             'cd: can\'t cd to' in command.output.lower(),
-            'does not exist' in command.output.lower()
```

Comment:
Wonder whether, now this list is getting longer, something like this would be better:

```python
if any(word in command.output.lower() for word in does_not_exist_list)
```